lib/utils/check-version.js

```diff
@@ -0,0 +1,41 @@
+const request = require("request-promise");
+const Logger = require.main.require("lib/logger/logger.js");
+const Errors = require.main.require("lib/response/error.js");
+
+function CheckVersion(pkg) {
```

Comment:
The vast majority of methods/functions (except for constructor functions) use `snake_case`. I suggest you rename the function to use that convention :)

BTW, isn't "check version" too ambiguous? One might think that this function returns the version stored in the `pkg` variable. Maybe "check for updates" would be more suiting?